### ISSUE TITLE
GET / POST annotations do not both show up in swagger

### ISSUE THREAD

**Issue by toumorokoshi** [toumorokoshi/flask-transmute#7]
annotating both GET / POST for the same paths don't seem to show up in swagger.

**Comment by aklatt1194** [comment]
The routes seem to register (I can curl them), but they are not showing up in swagger.

I have something like the following:

```
@route(bundle, paths='/bundle/{bundle_name}', methods=['GET'])
def bundle_get(bundle_name):
    return 'foo'

@route(bundle, paths='/bundle/{bundle_name}', methods=['POST'])
@annotate({'initial_version': StringType(required=True)})
def bundle_post(bundle_name, initial_version):
    return 'foo'

@route(bundle, paths='/bundle/{bundle_name}', methods=['DELETE'])
def bundle_delete(bundle_name):
    return 'foo'
```

Swagger is only showing the delete, however. If I combine them into a single function with methods=['GET', 'POST', 'DELETE'], then they all show up on swagger.

**Comment by toumorokoshi** [comment]
turns out this was a bug in transmute-core as well. fixed upstream in transmute-core 0.4.9. thanks!
